src/hooks/useUniswapXSwapCallback.ts

```diff
@@ -89,6 +89,7 @@ export function useUniswapXSwapCallback({
             fiatValues,
           }),
           ...analyticsContext,
+          time_to_sign: performance.mark('time-to-sign-uniswapx').startTime,
```

Comment:
why not just use the difference between Date.now() before sending the request on line 69 and after